### DISCUSSION TITLE
When will `react-i18next` peer dependency upgrade release will be made?

### DISCUSSION THREAD

**Discussion by thomasverleye** [sergiodxa/remix-i18next#197] [Q&A] [answered]
When upgrading `react-i18next` to `15.0.0` we're getting following error:
```
npm ERR! Could not resolve dependency:
npm ERR! peer react-i18next@"^[13](https://github.com/belgianmobileid/cra-b2b-portal-fe/actions/runs/10039829739/job/27744575664?pr=392#step:4:14).0.0 || ^14.0.0" from remix-i18next@6.1.0
```
I see a commit with the upgrade has already been done but was curious when this will be released 👀

**Comment by steuraa** [comment]
Was wondering the same thing, any possibility for a new release with the `15.0.0` support?

**Comment by sergiodxa** [comment, marked as answer]
Support for react-i18next was added on v6.1.1
